### DISCUSSION TITLE
Test du nouveau module de chauffage Nodon SIN-4-FP-21 - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#43] [Comments]
# Test du nouveau module de chauffage Nodon SIN-4-FP-21 - Haade.fr

Je vais tester ce module nodon fil pilote et surtout te montrer comment l’intégrer et le contrôler dans home assistant avec l’interface thermostat

https://haade.fr/fr/blog/test-nouveau-module-chauffage-zigbee-nodon-sin-4-fp-21-avec-fil-pilote-et-integration-home-assistant

**Comment by F5CUA** [comment]
Bonjour, et merci pour cette synthèse sur les modules de commande de fils pilote en Zigbee.
De mon coté je prévois une installation pour plus d'une dizaine de radiateurs déjà en fonctionnement. Mais contenu de l'investissement, je préfère faire quelques essais avant.
Mon premier objectif est de mieux gérer un système de délestage devenu obsolète au fil du temps à partir des données collectées sur un "Linky" récemment installé.
La sortie de câble Legrand offre les mêmes possibilités que les modules NodOn avec un avantage la fonction sortie de câble :)
Remarques : Avec ZHA aucune possibilité d’agir sur le fil pilote non plus. En revanche pas de problème avec Z2M
Les sorties de câbles pour radiateurs sont généralement "planquées" entre un mur et le radiateur métallique. Ce n'est pas un environnement favorable pour des communications radio.
Avantage aux modules que l'on peux placer dans une petite boite en dehors de l'écran formé par le radiateur.

Pensez à bien étudier votre réseau Zigbee.

A+

**Reply by haade-administrator** [reply]
Merci pour ces conseils très pertinents

**Comment by tazelbaze** [comment]
Bonjour et merci pour cet article très clair.
Est-ce qu'il existe des retours d'expérience avec Versatile Thermostat plutôt que le Generic ?
Merci

**Reply by haade-administrator** [reply]
Bonjour non pas de retour mais ça ne devrait pas poser de soucis car versatile thermostat est très complet, il faudra choisir les paramètres adéquats.

**Comment by zic700** [comment]
Bonjour, bel article merci !

Si on reste avec HA sans passer par Z2M, on a bien du on/off sans fil pilote effectivement. Dans mon cas ca pourrait suffire. Mais le ON correspond du coup au mode CONFORT et le OFF à off ou eco ?

**Comment by timautin** [comment]
Bonjour,
Je viens d'acheter 2 de ces modules pour intégration dans ZHA. J'ai le suivi conso, et les 3 modes suivants :

- off
- hors-gel
- confort

Confort - 1, confort - 2 et éco ne marchent pas, ni les mises à jour OTA (j'ai Installé le dernier firmware via l'appli Nodon).
J'ai contacté le support Nodon pour en savoir plus, à suivre...

**Reply by haade-administrator** [reply]
Les modes sont censés fonctionner, comme ça concerne 2 modules, regarde voir la connectiques et surtout la notice de tes radiateurs, pour les mises à jours peuvent être un peu plus tatillons.

**Reply by haade-administrator** [reply]
pour la mise à jour ota sous zha tu trouveras des infos ici: [forum hacf](https://forum.hacf.fr/t/maj-ota-pour-zha-en-2025/55692)

**Comment by timautin** [comment]
Ah super pour les maj OTA mes modules sont maintenant tous listé comme à jour, merci ! J'avais lu que c'était natif maintenant et qu'il n'y avait rien à faire, mais en effet pour Nodon il faut rajouter ça au configuration.yaml et redémarrer :

```
# Màj OTA ZHA
zha:
  zigpy_config:
    ota:
      enabled: true
      broadcast_enabled: true
      broadcast_initial_delay: 3600 # Délai avant la première notification OTA
      broadcast_interval: 21600 # Intervalle entre les diffusions
      extra_providers:
        - type: sonoff # Pour les appareils SONOFF/eWeLink
        - type: z2m # Pour les appareils couverts par Zigbee2MQTT
```

Pour les modes, je ne vois pas trop quoi faire d'autre, hormis tester sous Z2M :/ . Un des deux radiateurs est un Atlantic Solius Digital avec écran qui a le comportement que j'ai décrit, le 2e est un vieux modèle sans écran, compliqué de savoir s'il applique les consignes (il faut que je le laisse tourner plusieurs heures et regarder si la conso change).
J'essaierai de voir demain si le module Nodon envoie bien les ordres que le Solius attend (listés ici en p.7 : https://www.econology.fr/media/attachment/file/n/o/notice-utilisation-installation-solius-digital_1_.pdf)

**Comment by timautin** [comment]
Du coup verdict, le Nodon envoie bien les bons ordres, le problème était simplement qu'il faut mettre le radiateur en mode programmation externe, comme indiqué p13 du manuel. Tout fonctionne parfaitement :)

**Comment by alka79** [comment]
Bonjour,
Google amène ici lorsqu'on cherche des informations sur ces modules et c'est une bonne chose car l'article est bien détaillé.
J'ai acquis le cousin de chez Leroy Merlin dans la gamme Equation, marque Adeo, ref SIN-4-FP-21_EQU. Il est immédiatement reconnu en ZHA. Les 6 ordres sont là et la consommation aussi. Je n'ai pas les mise à jour firmware, mais du moment que ça marche ce n'est pas bien grave.
L'information de la compatibilité ZHA de ces modules est dans les commentaires mais ce serait cool de mettre l'article à jour sur ce point - ou renvoyer vers les commentaires.
Bonne continuation,
